packages/client/tests/functional/_utils/idForProvider.ts

```diff
@@ -0,0 +1,6 @@
+const sqlId = 'Int @id @default(autoincrement())'
+const mongoDbId = 'String @id @default(auto()) @map("_id") @db.ObjectId'
+
+export function idForProvider(provider: string): string {
```

Comment:
Thought this might be a useful helper for the cases, where we don't care about specifics of the id. Which, I assume will be most of the tests. What do you think?